libs/langchain/tests/unit_tests/tools/openapi/test_api_models.py

```diff
@@ -86,30 +78,38 @@ def http_paths_and_methods() -> List[Tuple[str, OpenAPISpec, str, str]]:
     return http_paths_and_methods
 
 
-@pytest.mark.parametrize(
-    "spec_name, spec, path, method",
-    http_paths_and_methods(),
-)
-def test_parse_api_operations(
-    spec_name: str, spec: OpenAPISpec, path: str, method: str
-) -> None:
+@pytest.mark.requires("openapi_schema_pydantic")
```

Comment:
i think we'll need to move it to integration tests with the requires marker right now

Reply:
ah shoot forgot to add to extended test, but why move to integration? are we gonna run extended test with pydantic v2 at some pt